pages/AddNewPage.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavigationBar } from "../components/Footer/NavigationBar.js";
+import AddNewForm from "../components/AddNew/AddNewForm.js";
+import useLocalStorageState from "use-local-storage-state";
+
+// Funktion
+
+export default function AddNewPage() {
+  const [appointmentList, setAppointmentList] =
+    useLocalStorageState("appointmentList");
+
+  function addAppointment(newAppointment) {
+    setAppointmentList([newAppointment, ...appointmentList]);
+  }
+
+  return (
+    <>
+      <HeadlineContainer>
+        <ProfileImage />
+        <Headline>Hej Alex!</Headline>
+      </HeadlineContainer>
+      <AddNewForm onAddAppointment={addAppointment} />
```

Comment:
wegen Konvention würde man das in Klammern hier "handleAddAppointment" nennen und dann dementsprechend in Zeile 12 auch :)